<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>591</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="fileOpenAction"/>
    <addaction name="fileNewAction"/>
    <addaction name="fileCloseAction"/>
    <addaction name="fileSaveAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>立體圖形</string>
     </property>
     <addaction name="actioncuboid"/>
     <addaction name="actioncylinder"/>
     <addaction name="actionnull"/>
    </widget>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="fileOpenAction">
   <property name="text">
    <string>打開</string>
   </property>
   <property name="toolTip">
    <string>打開</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="fileNewAction">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="fileCloseAction">
   <property name="text">
    <string>關閉</string>
   </property>
   <property name="toolTip">
    <string>關閉</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="fileSaveAction">
   <property name="text">
    <string>儲存</string>
   </property>
   <property name="toolTip">
    <string>儲存</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionand">
   <property name="text">
    <string>and</string>
   </property>
  </action>
  <action name="actionor">
   <property name="text">
    <string>or</string>
   </property>
  </action>
  <action name="actionnot">
   <property name="text">
    <string>not</string>
   </property>
  </action>
  <action name="linearAction">
   <property name="text">
    <string>直線</string>
   </property>
   <property name="toolTip">
    <string>直線</string>
   </property>
  </action>
  <action name="curveAction">
   <property name="text">
    <string>弧線</string>
   </property>
   <property name="toolTip">
    <string>弧線</string>
   </property>
  </action>
  <action name="planeAction">
   <property name="text">
    <string>平面圖形</string>
   </property>
   <property name="toolTip">
    <string>平面圖形</string>
   </property>
  </action>
  <action name="actioncuboid">
   <property name="text">
    <string>cuboid</string>
   </property>
  </action>
  <action name="actioncylinder">
   <property name="text">
    <string>cylinder</string>
   </property>
  </action>
  <action name="actionnull">
   <property name="text">
    <string>null</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
